apps/api/src/app/portfolio/portfolio.service.ts

```diff
@@ -1180,6 +1181,11 @@ export class PortfolioService {
                   new AccountClusterRiskSingleAccount(
                     this.exchangeRateDataService,
                     accounts
+                  ),
+                  new AllocationClusterRiskEmergingMarkets(
```

Comment:
Move to a new section:
`allocationClusterRisk: summary.ordersCount > 0 ? await this.rulesService.evaluate([ ... ])`